.changeset/rotten-readers-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aligent/cdk-static-hosting": major
```

Comment:
I don't think this is major 😅 Major would imply it breaks old versions but I don't think this would?